src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -588,6 +588,35 @@ def _encode_crl_distribution_points(backend, crl_distribution_points):
 }
 
 
+class _PasswordUserdata(object):
+    def __init__(self, backend, password):
+        self.backend = backend
+        self.password = password
+        self.called = 0
+        self.exception = None
+
+
+def _pem_password_cb(buf, size, writing, userdata_handle):
+    ud = backend._ffi.from_handle(userdata_handle)
```

Comment:
@reaperhulk interested in your thoughts here, IMO this shouldn't use the global `backend`, but instead it should directly use the global `ffi`?